src/js/components/RootView.jsx

```diff
@@ -1,78 +1,143 @@
+                            <ChooseOrientationComponent/>
+                        </FilterBox>
+
+                        <FilterBox headerText="Size">
+                            {sizeFilters.map((sizeFilters, idx) => {
+                                return <FilterValueWithCheckbox key={idx + "opt"} name={sizeFilters.name} amount={sizeFilters.amount} />
+                            }) }
+                        </FilterBox>
+
+                        <FilterBox headerText="Customer Rating">
+                            <FilterValueWithCheckbox name="85x55mm" amount="112" />
+                            <FilterValueWithCheckbox name="185x155mm" amount="167" />
+                        </FilterBox>
+
+                        <FilterBox headerText="Customer Rating">
+                            <p>Header</p>
+                            {ratingFilters.map((filterOption, idx) => {
+                                return <FilterValueWithRating key={idx + "opt"} fillStar={filterOption.fillStar}  notFillStar={filterOption.notFillStar} amount={filterOption.amount} />
+                            }) }
+
+                        </FilterBox>
+
+                        <FilterBox headerText="Favorite">
+                            {favoriteFilter.map((filterOption, idx) => {
+                                return <FilterValueWithCheckbox key={idx + "opt"} name={filterOption.name} amount={filterOption.amount} />
+                            }) }
+                        </FilterBox>
+
+                        <FilterBox headerText="Industry">
+                            {industryFilters.map((filterOption, idx) => {
+                                return <FilterValueWithCheckbox key={idx + "opt"} name={filterOption.name} amount={filterOption.amount} />
+                            }) }
+                        </FilterBox>
 
                     </asside>
-                    <div className="flex-item-3">
-                        <Comments author="Anne Droid" bodyText="Something stranege" />
-                        <OptionListChildren>
-                            <ItemRenderer name="one" />
-                            <ItemRenderer name="two" />
-                            <ItemRenderer name="three" />
-                            <ItemRenderer name="four" />
-                        </OptionListChildren>
-                    </div>
+                    <section className="flex-item-3">
+
+                        <StarRatingComponent fillStar={fillStar}  notFillStar={notFillStar} />
+
+
+                        <OptionList fillStar={fillStar}  notFillStar={notFillStar} productName="AutomobilTest" productDesc="90x50mm" >
+
+                        </OptionList>
+                        <div>
+                            <ul>
+                                {products.map((product, idx) => {
+                                    return <li className="productListItem">
+                                        <OptionListElement key={idx + "opt"}
```

Comment:
przesyłaj też cały obiekt product do OptionListElement (powinien się nazywać Product) i tam z niego skorzystaj. Dzięki takiemu rozwiązaniu jeżeli będziesz chciała dodać/usunąć jakieś pole w produkcie to zrobisz to w modelu danych i w samym komponencie